pipeline/lint.bash

```diff
@@ -6,11 +6,11 @@ here="$(dirname "$(readlink -f "$0")")"
 
 # TODO: Try to get rid of the NFS server IP.
 # We need to set the NFS server IP on Exoscale for the nfs-provisioner.
-NFS_SC_SERVER_IP=127.0.0.1 helmfile -e service_cluster -f "${here}/../helmfile/" lint
-NFS_WC_SERVER_IP=127.0.0.1 helmfile -e workload_cluster -f "${here}/../helmfile/" lint
+NFS_SC_SERVER_IP=127.0.0.1 helmfile -e service_cluster --selector chart=blackbox,ck8sdash -f "${here}/../helmfile/" lint
+#NFS_WC_SERVER_IP=127.0.0.1 helmfile -e workload_cluster -f "${here}/../helmfile/" lint
```

Comment:
You should be able to remove `NFS_{WC,SC}_SERVER_IP`.

Reply:
Removed